src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+designed to interpret Edisp, \code{EDispKernelMap}, which builds an Edisp kernel
+map, i.e., a 4-dimensional WCS map where at each sky-position is associated an
+Edisp kernel. The latter is a representation of the Edisp as a function of the
+true energy only thanks to the sub-class \code{EDispKernel}.
+
+\paragraph{Background model:}
+The BKG can be represented in \gammapy as either 1) a 2D map (\code{Background2D})
+of count rate normalised per steradians and energy at different
+energies and offset-angles or 2) as rate per steradians and energy, as a
+function of reconstructed energy and detector coordinates (\code{Background3D}).
+In the former, the background is expected to follow a radially symmetric shape,
+while in the latter, it can be more complex.
+
+
+\subsection{gammapy.maps}
+\label{ssec:gammapy-maps}
+The \code{gammapy.maps} sub-package provides classes that represent data
+structures associated with a set of coordinates or a region on a sphere. In
+addition it allows to handle an arbitrary number of non-spatial data
+dimensions, such as time or energy. It is organized around three types of
+structures: geometries, sky-maps and map axes, which inherit from the base
+classes \code{Geom}, \code{Map} and \code{MapAxis} respectively.
+
+The geometry object defines the pixelization scheme and map boundaries. It also
+provides methods to transform between sky and pixel coordinates. Maps consist
+of a geometry instance together with a Numpy data array containing the corresponding
+map values. Map axes contain a sequence of ordered values which define bins on
+a given dimension, spatial or not. Map axes can have physical units attached to
+them, as well as define non-linear spaced bins. All map classes support operations such
+as arithmetic operations with unit support, up- and downsampling along
+extra axes, interpolation, resampling of extra axes, interactive visualisation
+in notebooks and interpolation onto different geometries.
+
+The sub-package provides a uniform API for the FITS World Coordinate System
+(WCS), the HEALPix pixelization and region-based data structure
+(see Figure~\ref{ig*:minted:gp_maps}).
+
+\begin{figure}
+	\import{code-examples/generated/}{gp_maps}
+
+	\caption{
+        Using \code{gammapy.maps} to create a WCS, a HEALPix and a region
+		based data structures. The initialisation parameters include
+        consistently the positions of the center of the map, the pixel
+        size, the extend of the map as well as the energy axis definition.
+        The energy minimum and maximum values for the creation of the
+        \code{MapAxis} object can be defined as strings also specifying the
+        unit. Region definitions can be passed as strings following
+        the DS9 region specifications \url{http://ds9.si.edu/doc/ref/region.html}.
+        }
+    \label{ig*:minted:gp_maps}
+\end{figure}
+
+% itemize because it helps me write, could also just be paragraphs...
```

Comment:
the `subsubsection` creates a `\newline` between the 3. And there is not just before the paragraph of the `MapAxis`.
Maybe a `\paragraph{XXX:}` is better, and taking care that we have a `\newline` before the `MapAxis` paragraph...